Tests/Invoke-DockerPull.Tests.ps1

```diff
@@ -0,0 +1,56 @@
+Import-Module $PSScriptRoot/../Docker.Build.psm1
+Import-Module -Global -Force $PSScriptRoot/MockReg.psm1
+. "$PSScriptRoot\..\Private\Invoke-Command.ps1"
+
+Describe 'Pull docker images' {
+
+    BeforeAll {
+        $script:moduleName = (Get-Item $PSScriptRoot\..\*.psd1)[0].BaseName
+    }
+
+    $code = {
+        Write-Debug $Command
+        StoreMockValue -Key "pull" -Value $Command
+    }
+
+    BeforeEach {
+        Initialize-MockReg
+        Mock -CommandName "Invoke-Command" $code -Verifiable -ModuleName $script:moduleName
+    }
+
+    AfterEach {
+        Assert-MockCalled -CommandName "Invoke-Command" -ModuleName $script:moduleName
+    }
+
+    Context 'Docker pulls docker images' {
+
+        It 'pulls public docker image with image name only' {
+            # This test will most likely fail IRL because there isn't any latest tag available for this image
```

Comment:
What about finding another test case that doesn't fail IRL?

Reply:
Found iis image that has latest. ;)